specification/managementgroups/resource-manager/Microsoft.Management/preview/2018-03-01-preview/management.json

```diff
@@ -0,0 +1,1494 @@
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Management/checkNameAvailability": {
+      "post": {
+        "tags": [
+          "CheckNameAvailability"
+        ],
+        "operationId": "CheckNameAvailability",
+        "description": "Checks if the specified management group name is valid and unique",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/CheckNameAvailabilityParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/CheckNameAvailabilityResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CheckManagementGroupNameAvailability": {
+            "$ref": "./examples/CheckManagementGroupNameAvailability.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/getEntities": {
+      "post": {
+        "tags": [
+          "Entities"
+        ],
+        "operationId": "Entities_List",
+        "description": "List all entities (Management Groups, Subscriptions, etc.) for the authenticated user.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SkipTokenParameter"
+          },
+          {
+            "$ref": "#/parameters/SkipParameter"
+          },
+          {
+            "$ref": "#/parameters/TopParameter"
+          },
+          {
+            "$ref": "#/parameters/SelectParameter"
+          },
+          {
+            "$ref": "#/parameters/SearchParameter"
+          },
+          {
+            "$ref": "#/parameters/EntityFilterParameter"
+          },
+          {
+            "$ref": "#/parameters/EntityViewParameter"
+          },
+          {
+            "$ref": "#/parameters/GroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CacheControlHeader"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/EntityListResult"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "GetEntities": {
+            "$ref": "./examples/GetEntities.json"
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
@rajshah11 - can you confirm that this list operation is indeed also a long running operation?